CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0] - 2015-03-17
+### Changed
+* No more mapnik dependency by default. Instead all tile logic has been moved to [koop-tiles-plugin](https://github.com/koopjs/koop-tile-provider)
```

Comment:
Couple typos: you named it koop-tile-plugin and it's at https://github.com/koopjs/koop-tile-plugin